Noisily sorted.txt

```diff
@@ -0,0 +1,5 @@
+1
+N true RANDOM true 56 71 0 1
+N false SORTED 363 125 1 -1
+C 0 1
+C 1 2
```

Comment:
why do connections have to be specified redundantly? (personally I'd prefer inferring node pre- and post-connections from the connections themselves)

Reply:
Because it's easier to parse, as this is how data is stored internally.

Reply:
but it's much more cumbersome to use (and is it really that hard to just iterate through the connections and set the node's connections?)

Reply:
How is it more cumbersome to use?

Reply:
if you want to manually input things (yeah you can use the shuffle dialog, but typing it up can be pretty convenient; also, it's clearer to read as text)

Reply:
That's fair. Maybe v2 of the format will support that (currently there's v0 and v1, it's that lone number at the start).

Reply:
alright, mind if I tack that onto the Projects board thing? (speaking of which, think it might be a bit outdated)

I don't mind to try implementing it myself if you want